api/libproject/generate_appengine_web_xml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+CIRCLECI="${CIRCLECI:-false}"
+
+if [[ ! "$CIRCLECI" == "true" ]]; then
+  cd db
+  source vars.env
```

Comment:
It's in our repo, but that statement says to not use it if `CIRCLECI` is "true," so it will pull from circle's environment variables instead.